pango/Gir.toml

```diff
@@ -332,6 +331,29 @@ concurrency = "send+sync"
         name = "glyphs"
         const = true
 
+[[object]]
+name = "Pango.Language"
+status = "generate"
+    [[object.function]]
+    pattern = "get_(scripts|preferred)"
+    manual = true # probably needs annotation fixes upstream (array zero-terminated=1)
+    [[object.function]]
+    pattern = "(get_sample_string|includes_script|matches|to_string)"
+        [[object.function.parameter]]
+        name = "language"
+        const = true
+    [[object.function]]
+    name = "from_string"
+    # This only returns NULL when passing NULL
+        [[object.function.parameter]]
+        name = "language"
+        nullable = false
+        [object.function.return]
+        nullable = false
+    [[object.function]]
+    name = "to_string"
+    manual = true # avoids renaming the function to not cause an API break
```

Comment:
I would have `to_str()` autogenerated and implement `to_string()` manually

Reply:
I don't know why I did the opposite...